src/pages/Contributors/OrganizationSearch.js

```diff
@@ -90,7 +92,7 @@ const OrganizationSearch = ({
             options={options}
             renderInput={(params) => (
               <TextField
-                {...params}
+                {...params} 
```

Comment:
is the extra space needed?

Reply:
No, can be removed.

Reply:
ok go ahead and remove it when you have a moment then.  thanks